p2p/src/network/yamux/p2p_network_yamux_reducer.rs

```diff
@@ -318,21 +344,61 @@ impl P2pNetworkYamuxState {
                             });
                         }
                     }
+                    YamuxFrameInner::WindowUpdate { difference } => {
+                        if *difference < 0 {
```

Comment:
@vlad9486  what about `0` ? that case is not covered. The other condition uses `> 0` and this one `< 0`, which means there is no handling for `0`

Reply:
Yes, initially I wrote `<=`, but apparently the peer sends window update with zero if it wants to open the stream but not send data, so it sends flag SYN. Of course it could send data with zero length body, but it sends window update.